recipe/meta.yaml

```diff
@@ -82,12 +82,13 @@ outputs:
         - empy >=3.3,<3.4
         # static graphing
         - graphviz
+        - pillow
         # report-timings
         - pandas >=1.0,<2
         - matplotlib-base
         # tutorials
-        - pillow
-        - urllib3
+        - requests
+        # note: deliberately excluded pympler as main loop logging is only something us devs are interested in
```

Comment:
It's not just pympler we exclude, there's also sqlparse. This section should only contain the dependencies required for the "optional dependencies" listed in the `pip install` command above.